lib/engine/game/g_1817_de/meta.rb

```diff
@@ -8,15 +8,17 @@ module G1817DE
       module Meta
         include Game::Meta
 
-        DEV_STAGE = :prealpha
+        DEV_STAGE = :alpha
         DEPENDS_ON = '1817'
 
         GAME_DESIGNER = 'Scott Petersen'
         GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/1817DE'
         GAME_LOCATION = 'Germany'
         GAME_RULES_URL = {
-          '1817DE' =>
+          '1817DE Rule Differences' =>
                           'https://docs.google.com/document/d/1lH3TcQc6etptZDWMbX1oZe7ne2VbbinAYkX8W_lpu-c/edit',
```

Comment:
@scottredracecar I believe @jenf meant this link, it still seems to lock people out - https://docs.google.com/document/d/1lH3TcQc6etptZDWMbX1oZe7ne2VbbinAYkX8W_lpu-c/edit

Looks like we could use https://github.com/tobymao/18xx/wiki/1817DE instead?

Reply:
Yes, that makes more sense. Fixed it now.